methods.py

```diff
@@ -1,33 +1,38 @@
-from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.support.wait import WebDriverWait
 
 
 class Methods(object):
```

Comment:
Let's create instance of this wrapper in BasePage class, and use it via inheritance.

Reply:
Ok, but Nazar has created BasePage so he has to participate in this

Reply:
Work together on this in this PR.

Reply:
Donr